kedro-datasets/pyproject.toml

```
@@ -118,7 +118,7 @@ pandas = [
 pickle-pickledataset = ["compress-pickle[lz4]~=2.1.0"]
 pickle = ["kedro-datasets[pickle-pickledataset]"]
 
-pillow-imagedataset = ["Pillow~=9.0"]
+pillow-imagedataset = ["Pillow~=10.0"]
```

```suggestion
pillow-imagedataset = ["Pillow>=9.0"]
```
There's no reason to drop support for earlier versions. (In fact, I don't know why the lower bound is there, but maybe the dataset uses some feature that wasn't available earlier.)